### PR TITLE
Win condition artifacts minor fix

### PR DESCRIPTION
**Pull request by SimonHeimberg**
**Type of change**
minor enhancement

**Issue(s) closed**
Fixes -

### CODE REVIEW

File: data/scripting/win_conditions/artifacts.lua [hunk 1]
```diff
@@ -34,8 +34,7 @@ local r = {
       for x=0, map.width-1 do
          for y=0, map.height-1 do
             local field = map:get_field(x,y)
-            if field.immovable and field.immovable:has_attribute("artifact") then
-               -- This assumes that the immovable has size small or medium, i.e. only occupies one field
+            if field.immovable and field.immovable:has_attribute("artifact") and field.immovabe.fields[1] == field then
```
Comment (Noordfrees) [inline data/scripting/win_conditions/artifacts.lua]:
```suggestion
            if field.immovable and field.immovable:has_attribute("artifact") and field.immovable.fields[1] == field then
```

**Comment by hessenfarmer** [comment]
@SimonHeimberg please consider committing the suggestion so we can merge this

**Comment by hessenfarmer** [comment]
@SimonHeimberg will you fix the typo please?

**Comment by SimonHeimberg** [comment]
> @SimonHeimberg will you fix the typo please?

Ooops, sorry. I missed the previous comment. I wonder how I managed to not see the notification and that there is a comment on the list of my pr.
Thanks for the reminder. So here we go.

**Review by hessenfarmer — APPROVED** [review]

**Pull request merged**